CONTRIBUTING.md

```diff
@@ -42,7 +42,95 @@ you have written a test like this, make sure it does not fail often by
 uncommenting the code that sets a random seed in `tests/main.cpp` and running
 your test many times.
 
-Information on how to build and run the tests is in the main README.md file.
+### Install Dependencies
+
+<details open>
+<summary>Linux</summary>
+
+Use your distributions' package manager to install the required dependencies. The commands shown below are for Ubuntu/Debian.
+
+```bash
+$ sudo apt-get update
+$ sudo apt-get install libarmadillo-dev
```

Comment:
`cmake` is also a dependency, so that may be worth adding here.

Reply:
Ok, I am adding it to both the Linux and macOS steps then.